CMakeLists.txt

```diff
@@ -1,141 +1,32 @@
 cmake_minimum_required(VERSION 3.21.4)
 
-cmake_policy(SET CMP0028 NEW)
-cmake_policy(SET CMP0042 NEW)
-cmake_policy(SET CMP0048 NEW)
-cmake_policy(SET CMP0057 NEW)
-cmake_policy(SET CMP0076 NEW)
```

Comment:
All these policies are `NEW` by default since cmake 3.13.  And I don't see any sense in setting them rather then bumping a version.